jest.config.js

```diff
@@ -1,12 +1,7 @@
 module.exports = {
-  collectCoverageFrom: [
```

Comment:
Removing these will remove coverage of any untested files. It means we'll have no measurement of how many files are completely untouched by tests.

Reply:
@Gudahtt This is true, but then again, all the shards will run coverage for all the tests (instead of only those tests that are part of the shard). I guess we just have to take the performance hit there, as jest does not seem to support coverage generation only for a specific shard. I'll re-add this field to make sure we also have coverage for untested files.

Reply:
fixed, so I am resolving this